webapp/src/main/java/org/apache/atlas/Atlas.java

```diff
@@ -143,6 +144,10 @@ public static void main(String[] args) throws Exception {
             initElasticsearch();
         }
 
+        if (configuration.getString("atlas.authorizer.impl").equalsIgnoreCase("atlas")) {
```

Comment:
Do we need to throw any error in case other than atlas is present ?

Reply:
No